### PR TITLE
FEP-710: Update focus state

### PR DESCRIPTION
**Pull request by AndreiRosu**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=Or8mfQW)
# Description

The focus state of the following components have been changed to match the new style (yellow and black outline):

- Accordion
- Details
- Forms (text input, segment, select)
- Table (expandable details)
- Hero notification links

### CODE REVIEW

File: projects/canopy/src/lib/accordion/accordion-panel-heading/accordion-panel-heading.component.scss [hunk 1]
```diff
@@ -28,7 +28,7 @@
   width: 100%;
 
   &:focus-visible {
-    @include lg-inner-focus-outline();
+    @include lg-focus-outline;
```
Comment (elenagarrone) [inline projects/canopy/src/lib/accordion/accordion-panel-heading/accordion-panel-heading.component.scss:31]: The focus for the accordion doesn't look exactly right. I feel like it should be internal to the box 🤔 worth checking with Alex:
Screenshot 2023-04-25 at 09 42 29

The details component will be the same

Comment (AndreiRosu) [reply projects/canopy/src/lib/accordion/accordion-panel-heading/accordion-panel-heading.component.scss:31]: I've checked this with Alex and he said the focus outline should be kept outside of the box. So I've added a z-index for the focused element to fix this.

Comment (elenagarrone) [reply projects/canopy/src/lib/accordion/accordion-panel-heading/accordion-panel-heading.component.scss:31]: Perfect thank you 👍 I will deploy in a sec so I can double check the changes 👍

**Comment by elenagarrone** [comment]
@alexcanning is this spacing ok when the title is focused?
Screenshot 2023-04-26 at 13 32 47

**Comment by alexcanning** [comment]
image

**Comment by alexcanning** [comment]
There should be 24px padding between the Accordion header area and the content when expanded.

**Comment by AndreiRosu** [comment]
A top-padding of 24px has been added for the `active` accordion-header item, as requested. (https://github.com/Legal-and-General/canopy/pull/1181/commits/75edd5d69eee0d19201f291ff18557257fa30faa)

Screenshot 2023-04-27 at 09 38 50
Screenshot 2023-04-27 at 09 54 30

File: projects/canopy/src/lib/accordion/accordion-item/accordion-item.component.scss [hunk 2]
```diff
@@ -9,5 +9,6 @@
 
   &--active {
     display: block;
+    padding-top: var(--space-md);
```
Comment (elenagarrone) [inline projects/canopy/src/lib/accordion/accordion-item/accordion-item.component.scss:12]: I would add this to line 3 of this file. Also the same has to be done for the Details component.

Comment (AndreiRosu) [reply projects/canopy/src/lib/accordion/accordion-item/accordion-item.component.scss]: Done 
Screenshot 2023-04-27 at 15 59 48

**Review by elenagarrone — DISMISSED** [review]
Looking good to me 👍

**Review by pauleustice — DISMISSED** [review]
Great stuff, thanks @AndreiRosu!

**Review by pauleustice — APPROVED** [review]

**Review by elenagarrone — APPROVED** [review]
👏

**Pull request merged**